test-results.mdx

```diff
@@ -17,21 +18,22 @@ Test results are always displayed in the app. The test report section on the fro
 
 ## Test results in your CI/CD pipeline
 
-Once tests are executed, you will receive a comment in your GitHub / Azure DevOps pipeline which might look like the example below.
+Once tests are executed, you will receive a comment in your GitHub / Azure DevOps pipeline which looks like the example below.
 
-<Frame caption="Example of Octomind test results in a commit comment, screenshot 06/2023">
+<Frame caption="Example of Octomind test results in a commit comment, screenshot 09/2023">
   <img
     src="images/Octomind-test-results-commit-comments-GitHub-example.png"
-    alt="Example of Octomind test results in a commit comment"
+    alt="Example of Octomind test results in a github pull request comment"
   />
 </Frame>
 
-| column          | description                                                                                                                                         |
-| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
-| description     | Name of the test case                                                                                                                               |
-| execution state | Can either be running or executed. If a test case is on the list of test cases to be executed but not executed yet, it will not appear in this list |
-| status          | checkmark: successfuly executed; hourglass: test case failed and is under investigation; red cross: test case failed                                |
-| details         | link which takes you to the test case details (see below)                                                                                           |
+There will be a row for each test with the following information:
+
+| column      | description                                                                                                          |
+| ----------- | -------------------------------------------------------------------------------------------------------------------- |
+| description | Name of the test case                                                                                                |
+| status      | checkmark: successfuly executed; hourglass: test case failed and is under investigation; red cross: test case failed |
```

Comment:
I thought hourglass means "in progress of execution". Also there is the "broken heart" emoji which now means there was an error?

Reply:
Fixes, this was just how it was before